bpf/process/generic_calls.h

```diff
@@ -130,19 +123,11 @@ generic_process_event1(void *ctx, struct bpf_map_def *heap_map,
 		       struct bpf_map_def *map, struct bpf_map_def *tailcals,
 		       struct bpf_map_def *config_map)
 {
-	struct execve_map_value *enter;
 	struct msg_generic_kprobe *e;
 	struct event_config *config;
 	unsigned long a1;
 	int zero = 0;
-	bool walker = 0;
 	long ty, total;
-	__u32 ppid;
-
-	/* Preamble to setup context */
-	enter = event_find_curr(&ppid, &walker);
-	if (!enter)
-		return 0;
```

Comment:
This is a place where the execve_map_value is used, no? If the value is nil we return. From what I can tell, the value is nill if we cannot find a value matching our ktime and our (or our ancestors') pid in the execve map.

That being said, I think it's fine to remove the code. We are checking the same thing in `generic_process_filter` and return `PFILTER_CURR_NOT_FOUND` which will lead to a PFILTER_REJECT from its caller, but let's update the commit message to reflect that.

Reply:
done!